rotkehlchen/inquirer.py

```diff
@@ -819,8 +834,13 @@ def find_curve_pool_price(
         May raise:
         - RemoteError
         """
-        ethereum = self.get_evm_manager(chain_id=lp_token.chain_id)
-        ethereum.assure_curve_cache_is_queried_and_decoder_updated()  # type:ignore  # ethereum is an EthereumManager here
+        assert lp_token.chain_id in CURVE_CHAIN_IDS, f'{lp_token} is not on a curve support chain'
+        evm_manager = self.get_evm_manager(chain_id=lp_token.chain_id)
+        assert isinstance(evm_manager, CurveManagerMixin), f'{lp_token} is not on a curve support chain'  # noqa: E501
```

Comment:
Why do you need this? Doesn't it understand chain id is of type Curve chain id? Force it somehow above and please avoid this assertion